CHANGELOG.md

```diff
@@ -2,6 +2,95 @@
+  * Fixed a bug that prevented upgrade from starting if the PDB `expectedPods` count is less than the `minAvailable` count.
+  * Fixed an error condition when AKS attempts to remove the taint `disk.csi.azure.com/agent-not-ready=NoExecute` on node startup.  More details: [https://github.com/kubernetes-sigs/azuredisk-csi-driver/pull/2309](https://github.com/kubernetes-sigs/azuredisk-csi-driver/pull/2309)
+  * Addressed an issue related to node subnet `IPAM Invoker Add failed with error: Failed to allocate pool` and the associated [agentbaker release](https://github.com/Azure/AgentBaker/pull/5551).
+  * Added validation when a cluster migrates to CNI Overlay to block migration when there is a custom ip-masq-agent config in the kube-system namespace.  This prevents loss of connectivity during migration.  See the [AKS documentation](https://learn.microsoft.com/en-us/azure/aks/upgrade-aks-ipam-and-dataplane) for more information.
+
+* Component updates:
+  * Cilium v1.14 version from v1.14.18-241220 to v1.14.18-250107 (v1.14.18-1) to include a fix for cilium dual stack upgrades. On upgrades, cilium config changes bpf-filter-priority from 1 to 2 but is not cleaning up the old filters at the old priority and as a result impacts connectivity. This patch will fix this bug, see GH issue in cilium repo for more details https://github.com/cilium/cilium/issues/36172
+  * Update the Azure disk driver version to v1.30.6 on AKS Version 1.30+
+  * Update Azure File CSI driver version to v1.29.10 on AKS 1.28
+  * Update Azure File CSI driver version to v1.30.7 on AKS 1.29 and 1.30
+  * Update Azure File CSI driver version to v1.31.3 on AKS 1.31
+  * Update Azure Disk CSI driver to v1.29.12 on AKS 1.28, 1.29
+  * Update Azure Disk CSI driver to v1.30.7 on AKS 1.30, 1.31
+  * Update Azure Blob CSI driver to v1.23.10 on AKS 1.28, 1.29
+  * Update Azure Blob CSI driver to v1.24.6 on AKS 1.30, 1.31
+  * Update Workload Identity image version to v1.4.0
+  * CNS/CNI updated to v.1.6.18 which includes Cilium nodesubnet support
+  * Added Multi-Instance GPU support for standard_nc40ads_h100_v5
+  * Update the OMS image to v3.1.25-1
+  * Update secrets store driver to v1.4.7 and akv provider to v1.6.2.
+  * Update Container Insights to v3.1.25
+  * Updates the Retina basic image to v0.0.23 on Linux and Windows: [release notes](https://github.com/microsoft/retina/releases/tag/v0.0.23) 
+  * Update karpenter image version to 0.6.1-aks
+  * Update agentbaker version to v0.20250122.0
+  * Update Cilium v1.16 from v1.16.5-250108 to v1.16.5-250110 (v1.16.5-1) to include a fix for Cilium dual stack upgrades. This will fix [https://github.com/cilium/cilium/issues/36172](https://github.com/cilium/cilium/issues/36172). The following CVEs are included in [v1.16.5](https://github.com/cilium/cilium/releases/tag/v1.16.5)
+    * [CVE-2024-52529](https://nvd.nist.gov/vuln/detail/CVE-2024-52529)
```

Comment:
it's mentioned following CVEs at the end of the line but only one is listed here. can we check if there were more?

Reply:
nothing else is listed in desc or relnotes